app/javascript/components/server-components/SignupPage.tsx

```diff
@@ -77,22 +77,35 @@ export const SignupPage = ({
       }
       headerActions={<a href={Routes.login_path({ next })}>Log in</a>}
     >
-      <form onSubmit={(e) => void handleSubmit(e)}>
+      <form onSubmit={(e) => void handleSubmit(e)} className="flex flex-col gap-12">
         <SocialAuth />
         <Separator>
           <span>or</span>
         </Separator>
-        <section>
+        <section className="flex flex-col gap-8 pb-12">
```

Comment:
Added `pb-12` to restore the bottom padding originally provided by the [legacy global form styles](https://github.com/antiwork/gumroad/blob/4423c2252fe0fc2edd665fee909a3f094f5deaf7/app/javascript/stylesheets/_forms.scss#L299).